appeals/web/src/server/views/appeals/appeal/outcome-valid-date.njk

```diff
@@ -0,0 +1,55 @@
+{%- extends "app/layouts/app.layout.njk" -%}
```

Comment:
This template seems largely identical to `appeals/web/src/server/views/appeals/appeal/update-due-date.njk` - would it make sense to use that instead of creating a new one? Ideally we'll refactor this to a pattern template at some point, but that's out of scope for this PR.